sos/report/plugins/microk8s.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2023 Canonical Ltd.,
+#                    David Negreira <david.negreira@canonical.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, UbuntuPlugin
+
+
+class Microk8s(Plugin, UbuntuPlugin):
+
+    short_desc = 'The lightweight Kubernetes'
+    plugin_name = "microk8s"
+    profiles = ('container',)
+
+    packages = ('microk8s')
+
+    microk8s_cmd = "microk8s"
+
+    def setup(self):
+        microk8s_units = [
+            'snap.microk8s.daemon-apiserver-kicker',
+            'snap.microk8s.daemon-cluster-agent',
+            'snap.microk8s.daemon-containerd',
+            'snap.microk8s.daemon-k8s-dqlite',
+            'snap.microk8s.daemon-kubelite'
+        ]
+
+        for unit in microk8s_units:
+            self.add_journal(units=unit)
```

Comment:
You can simply call `self.add_journal(units=microk8s_units)`, see https://github.com/sosreport/sos/blob/main/sos/report/plugins/__init__.py#L2884 and below.